packages/lib/src/components/ApplePay/components/ApplePayButton.tsx

```diff
@@ -19,7 +21,8 @@ class ApplePayButton extends Component<ApplePayButtonProps> {
     render({ buttonColor, buttonType }) {
         /* eslint-disable jsx-a11y/no-static-element-interactions */
         return (
-            <div
+            <button
```

Comment:
It might be a good idea to add a `type="button"` to this and maybe an `aria-label` as well.

Reply:
Good idea, I added those two attributes 👍